tasks/sep/006-0-sc-changes/SignFromJson.s.sol

```diff
@@ -0,0 +1,523 @@
+        require(EIP1967Helper.getImplementation(proxies.SuperchainConfig).code.length != 0, "7101");
+
+        SuperchainConfig superchainConfigToCheck = SuperchainConfig(proxies.SuperchainConfig);
+        require(superchainConfigToCheck.guardian() == superchainConfigGuardian, "7200");
+        require(superchainConfigToCheck.guardian().code.length != 0, "7250");
+        require(superchainConfigToCheck.paused() == false, "7300");
+    }
+
+    function checkLivenessModule() internal view {
+        console.log("Running assertions on the LivenessModule");
+
+        // See sepolia.json for config values being verified:
+        // https://github.com/ethereum-optimism/optimism/pull/10224/files
+        require(livenessModule.version().eq("1.1.0"), "checkLivenessModule-000");
+        require(livenessModule.safe() == address(securityCouncilSafe), "checkLivenessModule-100");
+        require(livenessModule.livenessGuard() == address(livenessGuard), "checkLivenessModule-200");
+        require(livenessModule.livenessInterval() == 31536000, "checkLivenessModule-300");
+        require(livenessModule.minOwners() == 2, "checkLivenessModule-400");
+        require(livenessModule.thresholdPercentage() == 20, "checkLivenessModule-500");
+        require(livenessModule.fallbackOwner() == address(foundationSafe), "checkLivenessModule-600");
+    }
+
+    function checkLivenessGuard() internal view {
+        console.log("Running assertions on the LivenessGuard");
+
+        require(livenessGuard.version().eq("1.0.0"), "checkLivenesssGuard-000");
+        require(livenessGuard.safe() == address(securityCouncilSafe), "checkLivenessGuard-100");
+
+        // Each owner was recorded as live when the Guard was deployed, or was recorded as live during execution of
+        // this runbook.
+        address[] memory owners = securityCouncilSafe.getOwners();
+        for (uint256 i = 0; i < owners.length; i++) {
+            uint256 lastLive = livenessGuard.lastLive(owners[i]);
+            require(lastLive == 1713416868 || lastLive == block.timestamp, "checkLivenessGuard-201");
+        }
+    }
+
+    function checkDeputyGuardianModule() internal view {
+        console.log("Running assertions on the DeputyGuardianModule");
+
+        require(deputyGuardianModule.version().eq("1.0.0"), "checkDeputyGuardianModule-000");
+        require(deputyGuardianModule.deputyGuardian() == address(foundationSafe), "checkDeputyGuardianModule-100");
+        require(deputyGuardianModule.safe() == address(securityCouncilSafe), "checkDeputyGuardianModule-200");
+        require(
+            deputyGuardianModule.superchainConfig() == address(proxies.SuperchainConfig),
+            "checkDeputyGuardianModule-300"
+        );
+    }
+
+    function checkSecurityCouncilSafe() internal view {
+        console.log("Running assertions on the SecurityCouncilSafe");
+
+        // The SecurityCouncilSafe and FoundationSafe should have the same set of owners
```

Comment:
nit since this won't translate to mainnet
```suggestion
        // The SecurityCouncilSafe and FoundationSafe should have the same set of owners on Sepolia only
```